imagetest/cmd/wrapper/main.go

```diff
@@ -17,21 +17,22 @@ import (
 )
 
 func main() {
-	// These are placeholders until daisy supports guest attributes.
```

Comment:
If the placeholder comment is no longer accurate can we just delete all of the placeholder stuff and start relying solely on guest attributes to signal that testing is finished?

Reply:
I'm not sure what the original author of this comment had in mind with the serial log being the placeholder.

My idea for now was to have both guest attributes and the serial log be a condition for finishing the test, so that in corner cases where guest attributes might fail (for example if we need to manually set guest attributes to be true on a VM, though this doesn't seem to be an issue), the tests can continue to finish with the serial log.

After a few daily test runs of CIT, if guest attributes end up being as reliable as we hope they are, we can remove the serial log match.

Reply:
I'm ok keeping both methods for a while as long as we keep tracking it and clean it up as soon as we have enough data to make a decision.

Reply:
cool, I created a bug for tracking this as well

Reply:
Will we be able to see that data? Daisy doesn't prefer one signal method, so we might see a signal match from serial logs even though guest attributes would have worked depending on the race.

Reply:
I think for the first few days, not seeing any tests break from this change should be good enough.

I've added an additional log statement when the guest attribute is placed from the HTTP request, so we can confirm in the logs that this succeeded.

Reply:
We are not setting interval for metadata signal so it will be min interval = 6s, if we are not setting interval for serial then it will be 10s meaning metadata signal will always happen first.

Reply:
But we can disable serial signal and watch, then remove it entirely if when we are confident it's working as expected.